dist/docker/redhat/Dockerfile

```diff
@@ -33,12 +29,22 @@ ADD scyllasetup.py /scyllasetup.py
 ADD commandlineparser.py /commandlineparser.py
 ADD docker-entrypoint.py /docker-entrypoint.py
 ADD node_exporter_install /node_exporter_install
-# Install Scylla:
-RUN curl $SCYLLA_REPO_URL -o /etc/yum.repos.d/scylla.repo && \
-    yum -y install epel-release && \
+
+# Prepare layer for actual software install
+RUN sed -i -e 's/enabled=1/enabled=0/' /etc/yum/pluginconf.d/fastestmirror.conf && \
+    yum -y install epel-release hostname supervisor openssh-server openssh-clients rsyslog && \
     yum -y clean expire-cache && \
     yum -y update && \
-    yum -y install scylla-$VERSION hostname supervisor openssh-server openssh-clients rsyslog && \
+    yum clean all
```

Comment:
I think this `yum clean all` here is correct, but it disagrees with what the changelog says.

I think it's the changelog that should be fixed, not the code.